quantum/stm32/proton_c.mk

```diff
@@ -3,7 +3,10 @@
 # These are defaults based on what has been implemented for ARM boards
 AUDIO_ENABLE = yes
 RGBLIGHT_ENABLE = no
-BACKLIGHT_ENABLE = no
+
+ifneq ($(strip $(LED_MATRIX_ENABLE)), direct)
```

Comment:
what scenario does this cover? LED_MATRIX_ENABLE = direct doesnt seem supported then it turns backlight off?

Reply:
Ah shoot, forgot to update that section when that changed. I may not need this now, I'll have to revisit this aspect.